index.js

```diff
@@ -54,7 +67,18 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let total = 0;
+  for (let movie of movies){
+    total += movie.imdbRating 
+  }
+  console.log(total)
+  
+  return total > 0 ? total / movies.length : 0;
+}
```

Comment:
I love the ternary use.  Just remember to remove our `console.log()`s before pushing code.